docs/protocol/core/v3.0.rst

```diff
@@ -257,7 +257,7 @@ Node names
 ==========
 
 Except for the root node, each node in a hierarchy must have a name,
-which is a string of ASCII characters. To ensure consistent behaviour
+which is a string characters. To ensure consistent behaviour
```

Comment:
Maybe we need to say that it is a string of Unicode characters? I.e., we don't say anything about encoding, but we do say what the character set is?

Reply:
Btw happy to merge this PR and return to this. I think there is a broader discussion needed around character sets and encodings.

Reply:
There's an existing character set in a bullet point a few lines below:

> must consist only of characters in the sets `a-z`, `A-Z`, `0-9`, `-_.`

I agree we should return to this section after discussing character sets and encodings more broadly